geo/geojson.go

```diff
@@ -0,0 +1,86 @@
+package geo
+
+import (
+	"fmt"
+
+	"github.com/paulmach/orb"
+	"github.com/paulmach/orb/geojson"
+	"github.com/qri-io/starlib/util"
+	"go.starlark.net/starlark"
+)
+
+func parseGeoJSON(thread *starlark.Thread, _ *starlark.Builtin, args starlark.Tuple, kwargs []starlark.Tuple) (v starlark.Value, err error) {
+	var (
+		data      starlark.Value
+		dataBytes []byte
+		fc        *geojson.FeatureCollection
+	)
+	v = starlark.None
+
+	if err = starlark.UnpackArgs("parseGeoJSON", args, kwargs, "data", &data); err != nil {
+		return
+	}
+
+	switch val := data.(type) {
+	case starlark.String:
+		dataBytes = []byte(val)
+	default:
+		err = fmt.Errorf("parseGeoJSON only supports parsing string values")
```

Comment:
This error should sound more like a programming language error. Like, it's not even possible parse a non-string. How about "parseGeoJSON: invalid argument type, expected string".